pom.xml

```diff
@@ -612,11 +501,6 @@
                <version>${selenium.plugin.version}</version>
             </plugin>
             <plugin>
-               <groupId>org.codehaus.mojo</groupId>
-               <artifactId>failsafe-maven-plugin</artifactId>
```

Comment:
`org.codehaus.mojo` is outdated, we use `org.apache.maven.plugins` which is already in our projects.